jetstream_pt/third_party/llama/model_exportable.py

```diff
@@ -199,3 +204,44 @@ def forward(
       h = self.norm(h)
       output = self.output(h).float()
     return output
+
+  @staticmethod
+  def get_quantized_linear_weight_to_scaler_map():
```

Comment:
Would you consider to move these 3 static method to a quantization until class?

Reply:
Actually I intended to keep them as static methods of a model class, since they are 'attributes' of the model and are model specific. (e.g. gemma may have different maps)